docs/CODEOWNERS

```diff
@@ -35,11 +35,17 @@
 /client/sysinfo/ @koute
 /client/tracing/ @koute
 
+# Documentation audit
+/primitives/runtime @paritytech/docs-audit
+/primitives/arithmetic @paritytech/docs-audit
+# /primitives/core (to be added later)
+# /primitives/io (to be added later)
+
 # FRAME
-/frame/ @paritytech/frame-coders
-/frame/nfts/ @jsidorenko
+/frame/ @paritytech/frame-coders @paritytech/docs-audit
```

Comment:
Is it possible that the issue is that the @paritytech/docs-audit team is a subteam of the @paritytech/devrel team? 
![Screenshot 2023-05-11 at 9 49 47](https://github.com/paritytech/substrate/assets/13892277/76631bc4-49b6-4277-bee5-695fef2b95e5)

Reply:
I don't think that's an issue as @paritytech/frame-coders is also a subteam.
The team should have write access to the repo, which is does not have. Not sure though who can gran this access to the team. Any idea?
<img width="1103" alt="Screenshot 2023-05-18 at 10 00 54" src="https://github.com/paritytech/substrate/assets/3775927/d66da2f2-3ae0-4044-a877-fd2c4dd0d5fa">

<img width="1014" alt="Screenshot 2023-05-18 at 09 58 40" src="https://github.com/paritytech/substrate/assets/3775927/f9e648ff-6ce0-4e26-bdab-4a85eda6afbb">

Reply:
asked Daniel Artamendi, he will fix in due time.

Reply:
seems to be fixed now, thanks @wabkebab!